code/ui/manager/src/components/sidebar/Refs.tsx

```diff
@@ -145,7 +169,23 @@ export const Ref: FC<RefType & RefProps & { status?: State['status'] }> = React.
         {isExpanded && (
           <Wrapper data-title={title} isMain={isMain}>
             {state === 'auth' && <AuthBlock id={refId} loginUrl={loginUrl} />}
-            {state === 'error' && <ErrorBlock error={indexError} />}
+            {state === 'error' && (
+              <>
+                <ErrorBlock error={indexError} />
+                <Tree
+                  status={props.status}
+                  isBrowsing={isBrowsing}
+                  isMain={isMain}
+                  refId={refId}
+                  data={filteredIndex}
+                  docsMode={docsOptions.docsMode}
+                  selectedStoryId={selectedStoryId}
+                  onSelectStoryId={onSelectStoryId}
+                  highlightedRef={highlightedRef}
+                  setHighlightedItemId={setHighlightedItemId}
+                />
```

Comment:
If there's an error, it's possible that trying to render the Tree will fail, right?